### DISCUSSION TITLE
Tree Component Filter

### DISCUSSION THREAD

**Discussion by rakeshvermak** [primefaces/community#1563] [PrimeReact]
Hi, I am new to the PrimeReact library. I am facing an issue with the tree component. How can we filter node data based on multiple values selected by the user from a Multiselect in the Tree component of PrimeReact? Can we do it or not? Please suggest or provide me with a link where I can see the implementation.

**Comment by JL45312** [comment]
you can create a custom filter component and add it to the header using the header prop https://primereact.org/tree/#api.Tree.props.header - i do not know what this would visually look like.
